modules/kubernetes/pages/quickstart-with-gke.adoc

```diff
@@ -31,7 +42,7 @@ $ ./tg gke kustomize -s 1
 
 [source,console]
 ----
-$ kubectl apply -f deployment/tigergraph-gke.yaml
+$ kubectl apply -f deploy/tigergraph-gke.yaml
```

Comment:
I noticed this part isnt present for eks and aks?

Reply:
They should be at around line 40 - 45 for both documents